css/main.css

```diff
@@ -543,6 +542,56 @@ div#div-shop img {
   width: 100vw;
 }
 
+/* -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales -Contenedores de Secciones principales*/
+
+#div-pic-big {
```

Comment:
Can we add more descriptive ids and class names, `div-pic-big` does not say much